.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+**I'm submitting a ...**  (check one with "x")
+- [ ] bug report
+- [ ] feature request
+- [ ] support request
+
+**Current behavior:**
+<!-- Describe how the bug manifests. -->
+
+**Expected behavior:**
+<!-- Describe what the behavior would be without the bug. -->
+
+**Steps to reproduce:**
+<!-- If you are able to illustrate the bug or feature request with an example, please provide steps to reproduce -->
+
+**Related code:**
+
+```
+insert any relevant code here else remove this section
+```
+
+**Other information:**
+<!-- List any other information that is relevant to your issue. Stack traces, related issues, suggestions on how to fix, Stack Overflow links, forum links, etc. -->
+
+**Would you like to work on the issue?**
+
+Please let us know if you can work on it or the issue should be assigned to someone else.
```

Comment:
Maybe this should be wrapped in a HTML comment?

Reply:
do you mean to not display it in the template also?